packages/xrpl/snippets/src/claimPayChannel.ts

```diff
@@ -4,7 +4,7 @@ import {
   PaymentChannelCreate,
   PaymentChannelClaim,
   hashes,
-} from '../../dist/npm'
```

Comment:
This change should no longer be necessary after @JST5000's PR, right?

Reply:
@mvadari what PR? There is a follow up task on this PR to address `xrpl-local`

Reply:
cc @JST5000

Reply:
https://github.com/XRPLF/xrpl.js/pull/2168

Reply:
I removed xrpl-local altogether everywhere in this PR.